braindecode/datautil/preprocess.py

```diff
@@ -45,7 +45,9 @@ class Preprocessor(object):
     def __init__(self, fn, apply_on_array=True, **kwargs):
```

Comment:
```suggestion
    def __init__(self, fn, apply_on_array=True, dtype=None, **kwargs):
```

How about adding dtype then as an explicit param (with default None) to the constructor? And adding docstring saying it is only valid for apply_on_array? Or you think it would be worse this way?